lib/action_subscriber/publisher/async/in_memory_adapter.rb

```diff
@@ -65,6 +65,20 @@ def initialize
             create_and_supervise_consumer!
           end
 
+          def error_handler
+            # This is currently not memoized because the configuration might change by some other
+            # gem loaded after action subscriber starts publishing. Because this code path only
+            # executes when something terrible happens, I think this trade-off is fine.
+            handler = ::ActionSubscriber.configuration.async_publisher_error_handler
+
+            if !handler.respond_to?(:call) || handler.arity != 1
```

Comment:
It shouldn't be any harder to extract as long as it's scope is limited to the configuration object. Let's go ahead and do it now.